lib/api/storage/index.ts

```diff
@@ -12,16 +13,20 @@ const MAPPING = {
   [STORAGE.AZURE]: AzureStorage,
 }
 
-const storage = MAPPING[config.storage.type]()
-
-if (!storage) {
-  throw new Error('Storage is not implemented')
-}
+const storage = MAPPING[config.storage.type] ? MAPPING[config.storage.type]() : null
 
 export function uploadFile(file, folder: string): Promise<string> {
+  if (!storage) {
```

Comment:
is it a good idea that we don't restrict working without storage implementation?

Reply:
it is fine if we don't use storage in the project. the module should not launch an error to the app just for requiring it. But uploading a file without storage , maybe then it should throw an error.  I'll change it